README.md

```diff
@@ -2,58 +2,75 @@
+`tenpureto` is a tool that allows to easily bootstrap software projects from a
+template. Almost every programming language ecosystem provides some
+bootstrapping solution. What makes `tenpureto` different is simplicity with a
+reasonable amount of flexibility:
 
- * Templates can be split into independent features that you can selectively include,
- * Projects created from a template can be updated to include upstream changes,
- * Contributing to a template is as easy as contributing to any other project.
+* Templates can be split into independent features that you can selectively
+  include.
+* Projects created from a template can be updated to include upstream changes.
+* Contributing to a template is as easy as contributing to any other project.
 
 ## Installation
 
 ### macOS
 
 You can install `tenpureto` with [Homebrew](https://brew.sh):
 
-```sh
-$ brew tap rtimush/tap # you need it only once
-$ brew install tenpureto
+```sh 
+$ brew tap rtimush/tap # you need it only once $ brew install tenpureto
 ```
 
 ### Build from sources
 
 You need to get [Haskell Stack](https://haskellstack.org) first.
 
-If you are building on macOS, you will also need `icu4c`:
-```sh
-$ brew install icu4c
-```
-You will also need to let `Stack` know the location of the `icu4c` library. Add the following snippet to your `~/.stack/config.yaml`:
+If you are building on macOS, you will also need `icu4c`: 
+
+```sh 
+$ brew install icu4c 
+``` 
+
+You will also need to let `Stack` know the location of the `icu4c`
+library. Add the following snippet to your `~/.stack/config.yaml`: 
+
 ```yaml
 extra-lib-dirs:
-- /usr/local/opt/icu4c/lib
+  - /usr/local/opt/icu4c/lib
 
 extra-include-dirs:
-- /usr/local/opt/icu4c/include
+  - /usr/local/opt/icu4c/include 
 ```
 
-Once you have it installed, get `tenpureto` sources and run
-```sh
-$ stack build
+Once you have it installed, get `tenpureto` sources and run 
+
+```sh 
+$ stack build 
 ```
 
 ## Usage
 
-To create a new project run the following command:
-```sh
-$ tenpureto create --template <repository> <directory>
-```
+To create a new project run the following command: 
+
+```sh 
+$ tenpureto create --template <repository> <directory> 
+``` 
 where
- * `<repository>` — Git URL of a template repository (for GitHub you can just use `organization/repo`),
- * `<directory>` — where to create a project.
 
-You will be asked to choose a subset of template features you want to include, and to provide some variable values, and then the project will be created.
+* `<repository>` — Git URL of a template repository (for GitHub you can just
+    use `organization/repo`).
+* `<directory>` — where to create a project.
+
+You can also pass a local path (either to a clone of an online repo, or just a
+regular folder) instead of a Git URL.
```

Comment:
Let's mention that a clone must be made with a `--mirror` flag. Also, a regular folder that is not a git repository will not work, the template must be a git repo.

Reply:
Changed the wording - is that better?

Reply:
Yep, now sounds correct